deployment/articles/04-securing-liferay/03-service-access-policies.markdown

```diff
@@ -106,6 +107,26 @@ disallow certain API functions from being invoked, you can change the
 service access policies. Default service access policies are applied to all
 incoming requests, including unauthenticated requests. 
 
+The other 10 policies have to do with OAuth and 
+[JSON web services](/docs/7-2/frameworks/-/knowledge_base/f/invoking-json-web-services): 
+
+**OAUTH2_analytics.read/write:** Integrates with 
+[Liferay Analytics Cloud](https://www.liferay.com/products/analytics-cloud), 
+allowing it access to JSON web services. 
+
+**OAUTH2_everything/read/documents/userprofile/write:** The Everything policies
+grant access to all the JSON web services for various reasons. Everything is
+everything: all JSON web services (matches `*`). The others match method
+signatures appropriate to their description. For example, OAUTH2_everything.read
+matches all methods starting with `fetch`, `get`, `has`, `is`, or `search`. 
+
+**OAUTH_READ/WRITE:** These provide access to JSON web services via the OAuth
+1.0a plugin. 
+
+The default configuration opens access to all web services shipped with the
+system. Administrators should review the ones you want to use and disable the
+others. 
+
```

Comment:
I'm not sure about "opens access to all webservices shipped with the system". It sounds like by default access is open and the product is not "secure by default".

Maybe we could say something like "makes available corresponding scopes that provide access to all web services shipped with the system. The scopes must be assigned to OAuth 1 or OAuth 2 applications before they become usable."

Reply:
Haha. I was going for more scary. :-)